documentation/modules/configuring/proc-config-mirrormaker2-securing-connection.adoc

```diff
@@ -0,0 +1,257 @@
+
+The security options implemented for the source and target Kafka clusters must be compatible with the security options implemented for MirrorMaker 2.0.
+
+NOTE: In this procedure, the certificates generated by the Cluster Operator are used, but you can replace them by xref:installing-your-own-ca-certificates-str[installing your own certificates].
+You can also configure your listener to xref:kafka-listener-certificates-str[use a Kafka listener certificate managed by an external Certificate Authority].
+
+.Before you start
+Before starting this procedure, take a look at the link:{BookURLDeploying}#deploy-examples-{context}[example configuration files^] provided by Strimzi.
+They include examples for securing a deployment of MirrorMaker 2.0 using TLS or SCRAM-SHA-512 authentication.
+The examples specify internal listeners for connecting within a Kubernetes cluster.
+
+The examples also show the ACLs needed by MirrorMaker 2.0 to allow operations on the source and target Kafka clusters.
+
+.Prerequisites
+
+* Strimzi is running
+* Source and target Kafka clusters are available
+
+.Procedure
+
+. Configure two `Kafka` resources, one to secure the source Kafka cluster and one to secure the target Kafka cluster.
++
+You can add listener configuration for authentication and enable authorization.
++
+In this example, an internal listener is configured for a Kafka cluster with TLS encryption and authentication.
+Kafka `simple` authorization is enabled.
++
+.Example source Kafka cluster configuration with TLS authentication
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-source-cluster
+spec:
+  kafka:
+    version: {DefaultKafkaVersion}
+    replicas: 1
+    listeners:
+      - name: tls
+        port: 9093
+        type: internal
+        tls: true
+        authentication:
+          type: tls
+    authorization:
+      type: simple
+    config:
+      offsets.topic.replication.factor: 1
+      transaction.state.log.replication.factor: 1
+      transaction.state.log.min.isr: 1
+      default.replication.factor: 1
+      min.insync.replicas: 1
+      inter.broker.protocol.version: "{DefaultInterBrokerVersion}"
+    storage:
+      type: jbod
+      volumes:
+      - id: 0
+        type: persistent-claim
+        size: 100Gi
+        deleteClaim: false
+  zookeeper:
+    replicas: 1
+    storage:
+      type: persistent-claim
+      size: 100Gi
+      deleteClaim: false
+  entityOperator:
+    topicOperator: {}
+    userOperator: {}
+----
+
+. Create or update the `Kafka` resources.
++
+[source,shell,subs=+quotes]
+----
+kubectl apply -f _<kafka_configuration_file>_
+----
++
+The Cluster Operator creates the listeners and sets up the cluster and client certificate authority (CA) certificates to enable authentication within the Kafka cluster.
++
+The certificates are created with the same name as the Kafka resource.
```

Comment:
I'm not sure this makes sense. I guess you  might want to say something like

```suggestion
The certificates are created in the secret `<cluster-name>-cluster-ca-cert`.
```

The certificates do not really have a name and the Secret name differs.